### ISSUE TITLE
Adding task from taskwiki fails

### ISSUE THREAD

**Issue by MartyLake** [GothenburgBitFactory/taskwarrior#2131]
#### To report a bug...

* What command(s) did you run?
`Command used: task rc:/Users/mtalbot/.taskrc rc.confirmation=no rc.dependency.confirmation=no rc.recurrence.confirmation=no rc.json.array=off rc.bulk=0 rc.context= add description:'projectfiles : rename documentfiles' status:'pending' project:'Work' depends:`
* What did you expect to happen?
add a new task 'projectfiles : rename documentfiles' to the project 'Work'
* What actually happened?
`The expression could not be evaluated.`
* Paste the output of the ```task diag``` command.
```
task 2.5.1
   Platform: Darwin

Compiler
    Version: 4.2.1 Compatible Apple LLVM 10.0.0 (clang-1000.10.43.1)
       Caps: +stdc +stdc_hosted +LP64 +c8 +i32 +l64 +vp64 +time_t64
 Compliance: C++11

Build Features
      Built: Aug 24 2018 11:17:54
      CMake: 3.12.1
    libuuid: libuuid + uuid_unparse_lower
  libgnutls: 3.5.19
 Build type: Release

Configuration
       File: /[redacted for privacy]/.taskrc (found), 2103 bytes, mode 100644
       Data: /[redacted for privacy]/.task (found), dir, mode 40755
    Locking: Enabled
         GC: Enabled
    $VISUAL: vim +startinsert
     Server: taskwarrior.inthe.am:53589
         CA: /[redacted for privacy]/ca.cert.pem, readable, 2405 bytes
      Trust: ignore hostname
Certificate: /[redacted for privacy]/private.certificate.pem, readable, 2462 bytes
        Key: /[redacted for privacy]/private.key.pem, readable, 3660 bytes
    Ciphers: NORMAL
      Creds: inthe_am/[redacted for privacy]/************************************

Hooks
     System: Enabled
   Location: /[redacted for privacy]/.task/hooks
             (-none-)

Tests
      $TERM: xterm-256color (213x32)
       Dups: Scanned 652 tasks for duplicate UUIDs:
             No duplicates found
 Broken ref: Scanned 652 tasks for broken references:
             Task f7e78721-dd6f-4a03-bd00-a09596d56420 depends on nonexistent task: ["051ecc9f-69b0-4b1a-948e-41771790af5f"]
             Task 923ade0c-66d2-4b21-99af-30d71577dd65 depends on nonexistent task: ["196920f3-628e-41f2-b9dc-edc4a0abfbfa"]
             Task 196920f3-628e-41f2-b9dc-edc4a0abfbfa depends on nonexistent task: ["d2dd33f2-e075-4104-8ba9-008d370f9d23"]
             Task 8683fc3d-a331-491a-8079-54427fde7306 depends on nonexistent task: ["60887cd6-73dc-4629-b0a9-2b16287ff5fa"]
             Task 26c5198a-b96f-4ebd-b7e7-fb43f1bcadb1 depends on nonexistent task: ["c4b12606-a3c9-499f-a6bf-79d542d7b602"
             Task 26c5198a-b96f-4ebd-b7e7-fb43f1bcadb1 depends on nonexistent task: "4ad71151-b8f0-40fd-9811-83dbb199cc92"]

```

**Comment by MartyLake** [comment]
When removing 'description' as suggested in https://github.com/GothenburgBitFactory/taskwarrior/issues/1830 , it works. Maybe TW is not the correct repository to report to then ? @tbabej  ?

**Comment by tbabej** [comment]
I suspect this is related to a different issue (which I can't find right now), that basically has TW explode if description starts with a string containing a TW keyword - in this case 'project'.

Try description that does not have project... as the first word, i.e. "rename documentfiles"

**Comment by MartyLake** [comment]
I confirm that your workaround works !

**Comment by tbabej** [comment]
Duplicate of #1809, fixed in 2.5.2 and 2.6.0.
